oelint_adv/color.py

```diff
@@ -1,10 +1,30 @@
-_COLOR = None
+from colorama import Fore
+from colorama.ansi import AnsiCodes
 
+_COLOR = False
+_COLORS_BY_SEVERITY = {
+    "neutral": Fore.WHITE,
+    "info": Fore.GREEN,
+    "warning": Fore.YELLOW,
+    "error": Fore.RED,
+}
 
-def get_color():
+
+def get_colorize() -> bool:
+    """Returns weather or not the terminal output is to be colorized"""
     return _COLOR
 
 
-def set_color(value):
+def set_colorize(value: bool) -> None:
+    """Globally turn colored terminal output on/off"""
     global _COLOR
     _COLOR = value
+
+
+def get_color_by_severity(severity: str) -> AnsiCodes:
+    """Get an ANSI color code given a severity
+
+    Args:
+        severity (str): either `neutral`, `info`, `warning` or `error`
+    """
+    return _COLORS_BY_SEVERITY.get(severity, _COLORS_BY_SEVERITY["neutral"])
```

Comment:
That looks better, but still I don't like the assumption about what a "neutral" color is - I met way to many guys that prefer amber/green on black as their default... and they will get "white" with this one

Reply:
Ah dang, sure! Especially when people use white background in their terminals....
So, that what the `ResetAll` style is for, I guess. I misinterpreted that as an unused feature. Will change.